### PR TITLE
Rails forms and helpers

### PR DESCRIPTION
**Pull request by 8Bts**
In this branch, we created forms using helper methods of Rails. We have only 1 controller called `UsersController`, 1 model called `Users` and 2 view files for `new` and `edit` actions.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to new_user_path
```
Comment (mcrd25) [inline app/controllers/users_controller.rb]: This should redirect to an `edit)=_user_path`

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<% end %>
+<%= form_for @user do |f| %>
+  <%= f.label :username, 'Your username here :'%>
+  <%= f.text_field :username, placeholder: "name1234" %><br><br>
+  <%= f.label :username, 'Your Email here :'%>
+ <%= f.text_field :email, placeholder: "example@email.com" %><br><br>
+ <%= f.label :username, 'Your  Password here :'%>
+  <%= f.password_field :password, placeholder: "password" %><br><br>
+  <%= f.submit "Create" %>
+<% end %>
```
Comment (mcrd25) [inline app/views/users/new.html.erb:40]: You are missing the two previous commented out forms.
You ought to leave the two previous forms you created as comments.

See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form1.png) and this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form2.png) concerning the HTML form

And this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_form_tag2.png)

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Changes Required :mag: 

Hi Team! :wave: 

Great job! You simply have to address my two review comments and you should get the :+1: 
Sorry for the delay in the review. :disappointed: 


***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Review by rOluochKe — APPROVED** [review]
Hello Team!
### Well done, you got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**